### PR TITLE
EXLM-860 Solution Landing pages

### PR DESCRIPTION
**Pull request by ruhisingh1**
Please always provide the Jira Issue your PR is for, as well as test URLs where your change can be observed (before and after):

Jira ID: EXLM-860

Test URLs:

- Before: https://main--exlm--adobe-experience-league.hlx.live/en/docs/analytics
- After: https://feature-exlm-860--exlm--adobe-experience-league.hlx.live/en/docs/analytics

### CODE REVIEW

File: blocks/list/list.js [hunk 1]
```diff
@@ -1,22 +0,0 @@
-import { isDocPage } from '../../scripts/scripts.js';
```
Comment (ahmed-musallam) [inline blocks/list/list.js:1]: Do we no need the list block anymore? pretty sure it's needed for cases where there are other blocks within a list... can you elaborate on why this was deleted?

File: styles/styles.css [hunk 2]
```diff
@@ -1009,12 +1009,45 @@ main > div.browse-rail {
...
+  margin: 0;
+  width: 100%;
+}
 .docs.docs-solution-landing main .default-content-wrapper h1 {
   font-size: 44px;
   /* This font is not avaliable in spectrum so we are hardcoding it */
   line-height: var(--spectrum-line-height-xxxl);
   margin-top: 0;
 }
+.docs.docs-solution-landing main .default-content-wrapper h2 {
```
Comment (ahmed-musallam) [inline styles/styles.css:1022]: Is `default-content-wrapper` really necessary in all the rules introduced here? it seems the CSS might be specific enough without it.

**Review by ahmed-musallam — DISMISSED** [review]

File: blocks/list/list.js (hunk 1 above)
Comment (ruhisingh1) [reply blocks/list/list.js:1]: This block code isn't required. The file was present but empty initially. The code was added just for Solutions Landing page. So, removing it as we don't require this anymore.
https://github.com/adobe-experience-league/exlm/commit/2c1515e728fb1f51ae95aa77ef3d7c849b4c2557

Comment (ahmed-musallam) [reply blocks/list/list.js:1]: I see, you are ok to remove then :)

Comment (ahmed-musallam) [reply blocks/list/list.js:1]: Let's just make sure to test cases where there are code blocks in a list for example to make sure

File: styles/styles.css (hunk 2 above)
Comment (ruhisingh1) [reply styles/styles.css:1022]: yes @ahmed-musallam. Few lists and h1, h2, h3 on solution landing pages have different CSS compared to elements on rest of the page including mini-toc. And these are not within any specific block, so, we need this identifier to segregate the styles.

**Review by nitin-rachabathuni — DISMISSED** [review]

File: styles/styles.css [hunk 3]
```diff
@@ -849,22 +849,6 @@ main > div.browse-rail {
...
-    --exlm-font-size-h5: var(--spectrum-heading-size-xs);
-    --exlm-font-size-h6: var(--spectrum-heading-size-xxs);
-    --exlm-line-height-h1: var(--spectrum-line-height-xxxl);
-    --exlm-line-height-h2: var(--spectrum-line-height-xxl);
-    --exlm-line-height-h3: var(--spectrum-line-height-xl);
-    --exlm-line-height-h4: 23.4px;
-    --exlm-line-height-h5: 20.8px;
-    --exlm-line-height-h6: 18.2px;
-  }
-}
```
Comment (noormd08) [inline styles/styles.css:867]: Please keep the design tokens in the same place.

**Review by noormd08 — APPROVED** [review]

**Pull request merged**